crates/agglayer-config/src/storage.rs

```diff
@@ -104,6 +118,36 @@ impl From<StorageConfig> for StorageConfigHelper {
     }
 }
 
+fn default_db_path() -> PathBuf {
+    PathBuf::new().join(STORAGE_DIR)
+}
+
+pub(crate) fn normalize_path(path: &Path) -> PathBuf {
```

Comment:
I'd propagate the rustdoc from the cargo [code](https://github.com/rust-lang/cargo/blob/edd36eba5e0d6e0cfcb84bd0cc651ba8bf5e7f83/crates/cargo-util/src/paths.rs#L76-L83) that you shared maybe, at least the first line `Normalize a path, removing things like . and ..`